### PR TITLE
Clarify post training quantization behaviour

### LINKED ISSUE #2725: confusion in Tutorials >  Quantization Recipe
In 3. Post Training Static Quantization
" This method converts both the weights and the activations to 8-bit integers beforehand so there won’t be on-the-fly conversion on the activations during the inference, as the dynamic quantization does, **hence improving the performance significantly.**"

but I think the performance of Post Training Static Quantization is weaker than Post Training dynamic quantization since dynamic quantization can calibrate for each input.

cc @jerryzh168 @z-a-f @vkuzo

### PR DESCRIPTION
**Pull request by jmarintur**
Fixes #2725

## Description
This PR clarifies post training quantization behaviour

## Checklist

- [x] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [x] Only one issue is addressed in this pull request
- [x] Labels from the issue that this PR is fixing are added to this pull request
- [x] No unnecessary issues are included into this pull request.

### CODE REVIEW

File: recipes_source/quantization.rst [hunk 1]
```diff
@@ -81,7 +81,7 @@ The full documentation of the `quantize_dynamic` API call is `here <https://pyto
 3. Post Training Static Quantization
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This method converts both the weights and the activations to 8-bit integers beforehand so there won't be on-the-fly conversion on the activations during the inference, as the dynamic quantization does, hence improving the performance significantly.
+This method converts both the weights and the activations to 8-bit integers beforehand so there won’t be on-the-fly conversion on the activations during the inference, as the dynamic quantization does. While post-training static quantization can significantly enhance inference speed and reduce model size, this method may degrade the original model's performance.
```
Comment (jerryzh168) [inline recipes_source/quantization.rst:84]: makes sense, can you clarify a little bit more, e.g.:
```
this method may degrade the original model's accuracy more compared to dynamic quantization. 
```

Comment (usteiner9) [reply recipes_source/quantization.rst]: Like it

Comment (jmarintur) [reply recipes_source/quantization.rst]: Hi @jerryzh168, I've just pushed a few changes, please let me know if you have any other suggestions. Thank you!

Comment (jerryzh168) [reply recipes_source/quantization.rst]: @jmarintur "Converting weights and activation functions to 8-bit integers can slightly alter the network's behavior and activation responses, leading to such variations in accuracy." this is true for any quantization method, so probably better to remove I feel

File: recipes_source/quantization.rst [hunk 2]
```diff
@@ -81,7 +81,7 @@ The full documentation of the `quantize_dynamic` API call is `here <https://pyto
 3. Post Training Static Quantization
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This method converts both the weights and the activations to 8-bit integers beforehand so there won't be on-the-fly conversion on the activations during the inference, as the dynamic quantization does, hence improving the performance significantly.
+This method converts both the weights and the activations to 8-bit integers beforehand so there won’t be on-the-fly conversion on the activations during the inference, as the dynamic quantization does. While post-training static quantization can significantly enhance inference speed and reduce model size, this method may degrade the original model's accuracy. Converting weights and activation functions to 8-bit integers can slightly alter the network's behavior and activation responses, leading to a reduction in the model's original effectiveness.
```
Comment (jerryzh168) [inline recipes_source/quantization.rst:84]: nit: "this method may degrade the original model's accuracy" --> "this method may degrade the original model's accuracy more compared to post training dynamic quantization"

Comment (jmarintur) [reply recipes_source/quantization.rst]: Hi @jerryzh168, I've just updated the text following your suggestions. Thank you!

**Review by jerryzh168 — APPROVED** [review]
thanks!

**Pull request merged**